src/shipit_uplift/tests/conftest.py

```diff
@@ -11,9 +11,12 @@
 import os
 import pickle
 import pytest
+import subprocess
```

Comment:
Need to remove unused imports and `WORKER_MODULE` too

Reply:
Thanks, I was waiting for taskcluster to explode for me.